content/release/10.3.md

```diff
@@ -0,0 +1,1159 @@
+---
+Title: FRRouting Release 10.3
+Release: 10.3
+GitHub: https://github.com/FRRouting/frr/releases/tag/frr-10.3
+Date: 2025-03-18
+---
+
+FRRouting Release 10.3 Available for Download
+=============================================
+
+We are pleased to announce FRR release 10.3.
+
+FRR 10.3 brings a long list of enhancements and fixes with 1056 commits from 68 developers.
+
+Thanks to all contributors!
+
+Debian Packages - https://deb.frrouting.org
+
+RPM Packages - https://rpm.frrouting.org
+
+Snaps - https://snapcraft.io/frr
+
+Docker - [quay.io/frrouting/frr](https://quay.io/repository/frrouting/frr/manifest/sha256:f1b62a828583ed35012c754a5746568632afd324f1467735c50277b75e99a17b)
+
+## Release Overview
+
+### New Features Highlight:
+
+- Lua 5.4 support
+- Fixed CVE-2024-55553
```

Comment:
don't we have a CVE page that should be updated too?

Reply:
Right, added a link.